docs/.vitepress/config.ts

```diff
@@ -172,19 +172,19 @@ const SidebarGuide: DefaultTheme.SidebarItem[] = [
         link: '/integrations/',
       },
       ...Integrations,
+      {
+        text: 'Examples',
+        link: 'https://github.com/unocss/unocss/tree/main/examples',
```

Comment:
```suggestion
        link: '/integrations/#examples',
```

Maybe we should stop directly linking to the source